.travis.yml

```diff
@@ -28,4 +28,5 @@ install:
   - python setup.py install
 
 script:
-  - pytest
+  - travis_wait 30 pytest nifty/tests/test_nifsSort.py
```

Comment:
Not sure why the wait is required here. Can the functionality be achieved through other means? Typically one expects `pytest` to run all the tests in the `tests` directory whereas this approach requires explicit specification of test files.

Reply:
Hi Adrian, thanks so much for the feedback, sorry it took me so long to rework this.

I believe I've fixed the `wait` problem in test_nifsSort.py. 

The `-s` in the second call to pytest comes up because IRAF/PyRAF tasks still listen to stdin even when run in full automatic mode and error out (I believe) because they try to read from it during pyest. I see two possible solutions:

- Pass an empty txt file as stdin to IRAF tasks like so (might be safer that way to not let accidental stdin get through. Although it would require going through many, many IRAF tasks and making sure stdin is set up properly.):
```python
# Set up/prepare IRAF.
    with open('tmp.txt', 'w') as f:
        pass

    iraf.gemini(Stdin="tmp.txt")
    iraf.nifs(Stdin="tmp.txt")
    iraf.gnirs(Stdin="tmp.txt")
    iraf.gemtools(Stdin="tmp.txt")
```

Or pass the `-s` flag to pytest as before:

- `pytest -s <test_file>`

Maybe I'll mark this as a new issue to fully fix at some point.